lib/data/types.json

```diff
@@ -11,13 +11,13 @@
     ],
     "relief": [
       { "de": "nicht relevant", "fr": "pas important", "code": "unknown" },
+      { "de": "Kuppenlage", "fr": "sur une butte", "code": "kup" },
```

Comment:
These changes should be removed from the PR or we should sort `relief` data in PostgreSQL before exporting to JSON.